tutorials/3d/environment_and_post_processing.rst

```diff
@@ -59,65 +125,139 @@ Background
-
-Ambient Light
+- **Sky** lets you define a background sky material (see below). Objects in the
+  scene will reflect this sky material and absorb ambient light from it.
+- **Canvas** displays the 2D scene as a background to the 3D scene.
+- **Keep** does not draw any sky, keeping what was present on previous frames
+  instead. This improves performance in purely indoor scenes, but creates a
+  "hall of mirror" visual glitch if the sky is visible at any time.
+
+Sky materials
 ^^^^^^^^^^^^^
 
-Ambient (as defined here) is a type of light that affects every piece of geometry
-with the same intensity. It is global and independent of lights that might be
-added to the scene.
+When using the **Sky** background mode (or the ambient/reflected light mode is
+set to **Sky**), a Sky resource becomes available to edit in the Environment
+resource. Editing this subresource allows you to create a SkyMaterial resource
+within the Sky.
+
+There are 3 built-in sky materials to choose from:
+
+- **PanoramaSkyMateiral:** Use a 360 degree panorama sky image (2:1 aspect ratio
+  recommended). To benefit from high dynamic range, the panorama image must be
+  in an HDR-compatible format such as ``.hdr`` or ``.exr`` rather than a
+  standard dynamic range format like PNG or JPEG.
+- **ProceduralSkyMaterial:** Use a procedurally generated sky with adjustable
+  ground, sun, sky and horizon colors. This is the type of sky used in the
+  editor preview. The sun's position is automatically derived from the
+  DirectionalLight3Ds present in the scene.
+- **PhysicalSkyMaterial:** Use a physically-based procedural sky with adjustable
+  scattering parameters. The sun's position is automatically derived from the
+  first DirectionalLight3Ds present in the scene. PhysicalSkyMaterial is
+  slightly more expensive to render compared to ProceduralSkyMaterial.
+
+Panorama sky images are sometimes called HDRIs (High Dynamic Range Images).
+You can find freely licensed HDRIs on `Poly Haven <https://polyhaven.com/hdris>`__.
+
+If you need a custom sky material (e.g. for procedural clouds), you can
+create a custom :ref:`sky shader <doc_sky_shader>`.
+
+Ambient light
+^^^^^^^^^^^^^
 
-There are two types of ambient light: the *Ambient Color* (which is a constant
-color multiplied by the material albedo) and then one obtained from the *Sky*
-(as described before, but a sky needs to be set as background for this to be
-enabled).
+Ambient (as defined here) is a type of light that affects every piece of
+geometry with the same intensity. It is global and independent of lights that
+might be added to the scene. Ambient light is one of the two components of
+image-based lighting. Unlike reflected light, ambient light does not vary
+depending on the camera's position and viewing angle.
+
+There are several types of ambient light to choose from:
+
+- **Background:** Source ambient light from the background sky (default).
```

Comment:
In background mode the source will be whatever the ``Environment.background`` is set to. If the background is set to sky, it will use sky, but if the background is set to clear color it will use the clear color